### PR TITLE
Progress Report 2017 October and November

### LINKED ISSUE #65: Progress Report 2017 October
Double the progress report, double the ~~torture~~ FUN!

RFC will be open from 2017-11-25 08:00-0400 to 2017-12-10 23:59-0400.
Only PRs that were *merged* between 2017-10-01 and 2017-10-31.

### LINKED ISSUE #66: Progress Report 2017 November
Double the progress report, double the ~~torture~~ FUN!

RFC will be open from 2017-11-25 08:00-0400 to 2017-12-10 23:59-0400.
Only PRs that were *merged* between 2017-11-01 and 2017-11-30.

### PR DESCRIPTION
**Pull request by anodium**
Sorry for the delay, had some personal issues this past month that were preventing me from finishing this article.

Things that are still missing:

- [ ] Screenshots and graphs
- [ ] Banner
- [ ] Forum topic and metadata

Fixes #65
Fixes #66

### CODE REVIEW

**Comment by anodium** [comment]
Review time! :postal_horn:

- [ ] @bunnei
- [ ] @Subv
- [ ] @jroweboy

File: site/content/entry/citra-progress-report-2017-october-november.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+One of the most requested features for Citra, second only to controller support,
+has been enlarging the window to cover the entirety of the screen. After almost
+a year of requests on both the Citra Discourse and Discord servers, [Styleoshin](https://github.com/Styleoshin)
+has finally delivered!
+
+{{< figure src="/images/entry/citra-progress-report-2017-october-november/fullscreen.png" 
+    title="No more pesky window décor!" >}}
+
+Simply go into `View → Fullscreen`, or just strike the Alt + Return keycombo, and
```
Comment (Hexagon12) [inline site/content/entry/citra-progress-report-2017-october-november.md]: "just strike the _Alt + Return_ keycombo"

This is F11 in Windows.

Comment (jroweboy) [reply site/content/entry/citra-progress-report-2017-october-november.md]: both works iirc (i can double check later)

File: site/content/entry/citra-progress-report-2017-october-november.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    title="A tiny portion of the Home Menu Themes CIA" >}}
+
+For the `am` side of streaming installations, the way it handles a request to
+install an application is by creating and then giving the requester a handle to
+a virtual file. This leaves most of the busy work to `am`, while the app only needs
+to worry about writing the CIA file into that handle.
+
+[shinyquagsire23](https://github.com/shinyquagsire23) reimplemented the entirety
+of how Citra handles NCCH files, which quickly paved the way to implementing `am`
+in Citra. This allowed users to use FBI and the Home Menu to install and run
```
Comment (Subv) [inline site/content/entry/citra-progress-report-2017-october-november.md]: we can't run applications from the HM yet

File: site/content/entry/citra-progress-report-2017-october-november.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+date = "2018-01-10T13:11:00-04:00"
+title = "Citra Progress Report - 2017 October and November"
+tags = [ "progress-report" ]
+author = "anodium"
+forum = 0
++++
+
+Past the red leaves and into the icy tundra in the midst of winter comes the present.
+This time we'll be covering not one, but two months of progress! With a great many
+changes all around, this article is going to be packed to the brim with all the
```
Comment (Leo626) [inline site/content/entry/citra-progress-report-2017-october-november.md]: > With a great many changes all around

With many great changes all around

File: site/content/entry/citra-progress-report-2017-october-november.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+new goodies that have come to Citra those crimson months.
```
Comment (Leo626) [inline site/content/entry/citra-progress-report-2017-october-november.md]: > come to Citra those crimson months

come to Citra _during_ those crimson months

Not sure what crimson means in this context.

Comment (anodium) [reply site/content/entry/citra-progress-report-2017-october-november.md]: Autumn in temperate zones usually looks like this: 


Very red, as you might tell. :stuck_out_tongue:

Comment (Leo626) [reply site/content/entry/citra-progress-report-2017-october-november.md]: Oh now I get it 😅

**Comment by anodium** [comment]
Closed in favour of #73.

**Pull request closed without merging**